src/Psy/Shell.php

```diff
@@ -302,6 +305,7 @@ public function doRun(InputInterface $input, OutputInterface $output)
         // }
 
         $this->output->writeln($this->getHeader());
+        $this->output->writeln($this->getVersionCheck());
```

Comment:
this probably doesn't need to write the output, since you already did it inside `getVersionCheck`?